jobs/build/advisories/Jenkinsfile

```diff
@@ -180,6 +237,42 @@ node {
                 }
                 echo "out: ${out}"
             }
+            stage("create & attach Extras placeholder to advisory") {
```

Comment:
aesthetics issue: i don't really want all these giant stage names :) would you mind just putting all of these under "add placeholder bugs to advisories"?

Reply:
done :heavy_check_mark: